pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -213,6 +213,7 @@ message CommandConnect {
 message CommandConnected {
 	required string server_version = 1;
 	optional int32 protocol_version = 2 [default = 0];
+	optional int32 max_message_size = 3 [default = 5242880];
```

Comment:
For this kind of numbers, I'd leave the default empty and let the client/broker assume the 5MB default. 

In particular, if the default is in Java code, we can put in a constant shared between client and broker and keep it in a single place

Reply:
Yeah, you are right. will fix it